.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  root: true,
+  extends: 'eslint:recommended',
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    webextensions: true
+  },
+  globals: {
+    Components: false,
+    Services: false
+  },
+  rules: {
+    'eqeqeq': 'warn',
+    'no-var': 'warn',
+    'prefer-const': 'warn'
+  }
+};
```

Comment:
The one "gotcha" with these files is that we don't have a root level package.json file, so we aren't installing ESLint locally, and assume that the user would have it installed globally (assuming one wanted to run `$ eslint .`)